CoffeeChat/Controller/EditInterestViewController.swift

```diff
@@ -0,0 +1,195 @@
+        Interest(name: "Reading", categories: "lorem, lorem, lorem, lorem, lorem", image: "reading"),
+        Interest(name: "Sustainability", categories: "lorem, lorem, lorem, lorem, lorem", image: "sustainability"),
+        Interest(name: "Technology", categories: "lorem, lorem, lorem, lorem, lorem", image: "tech"),
+        Interest(name: "Travel", categories: "lorem, lorem, lorem, lorem, lorem", image: "travel"),
+        Interest(name: "TV & Film", categories: "lorem, lorem, lorem, lorem, lorem", image: "tvfilm")
+    ]
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .backgroundLightGreen
+
+        interestsTableView.delegate = self
+        interestsTableView.dataSource = self
+        interestsTableView.register(OnboardingTableViewCell.self, forCellReuseIdentifier: OnboardingTableViewCell.reuseIdentifier)
+        interestsTableView.isScrollEnabled = true
+        interestsTableView.clipsToBounds = true
+        interestsTableView.backgroundColor = .none
+        interestsTableView.allowsMultipleSelection = true
+        interestsTableView.bounces = false
+        interestsTableView.showsHorizontalScrollIndicator = false
+        interestsTableView.showsVerticalScrollIndicator = false
+        interestsTableView.separatorStyle = .none
+        interestsTableView.sectionFooterHeight = 0
+        view.addSubview(interestsTableView)
+
+        interestsTableView.snp.makeConstraints { make in
+            make.leading.trailing.equalTo(view.safeAreaLayoutGuide).inset(39)
+            make.top.bottom.equalTo(view.safeAreaLayoutGuide)
+        }
+    }
+
+    // MARK: - Data Management
+    private func getInterestFromIndexPath(_ indexPath: IndexPath) -> Interest {
+        return indexPath.section < yourInterests.count
+            ? yourInterests[indexPath.section]
+            : moreInterests[indexPath.section - yourInterests.count]
+    }
+
+}
+
+// MARK: - UITableViewDelegate
+extension EditInterestViewController: UITableViewDelegate {
+
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let interest = getInterestFromIndexPath(indexPath)
+        if indexPath.section < yourInterests.count { // From Top Section
+            yourInterests.removeAll(where: { $0.name == interest.name })
+            moreInterests.append(interest)
+            moreInterests.sort(by: {$0.name < $1.name })
+        } else { // Bottom Section
+            moreInterests.removeAll(where: { $0.name == interest.name })
+            yourInterests.append(interest)
+            yourInterests.sort(by: {$0.name < $1.name })
+        }
+        tableView.reloadData()
+    }
+}
+
+// MARK: - UITableViewDataSource
+extension EditInterestViewController: UITableViewDataSource {
+
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return 1
```

Comment:
why is the`numberOfRows` 1?

Reply:
The way I added spacing between the tableview cells was to make sections for each cell and control the size of the empty header. I couldn't find any other ways to control the row spacing within a section other than to edit the cells themselves, but that would've also affected the Onboarding Views